src/sections/flexbox.ts

```diff
@@ -64,9 +64,9 @@ export const flexbox: MethodFn = () => [
 
   // https://github.com/tachyons-css/tachyons/blob/master/src/_flexbox.css#L25
```

Comment:
This comment might not be worth including any longer now that we don't match the rules of Tachyons.

Reply:
Yep! Will address.